ansible_base/resource_registry/apps.py

```diff
@@ -67,15 +67,29 @@ def initialize_resources(sender, **kwargs):
             r_type.save()
 
 
+def connect_resource_signals(sender, **kwargs):
```

Comment:
passing `sender` or `**kwargs` doesn't make sense to me here. Even if you wanted to use it later, it's not really the right sender to use for these.